docs/configuration.md

```diff
@@ -42,6 +42,22 @@ module.exports = {
 };
 ```
 
+JS with type checking (eg. `checkJs: true` in `tsconfig.json`):
```

Comment:
Maybe we can use this  `DOCUSAURUS_CODE_TABS` comment https://github.com/prettier/prettier/blob/948b3af232d9be1cf973a143382cad3ebe33725a/docs/install.md?plain=1#LL8C21-L8C21